test/spec/Feature/Query/JsonOperatorSpec.hs

```diff
@@ -103,7 +103,7 @@ spec actualPgVersion = describe "json and jsonb operators" $ do
 
       it "can get array of objects" $ do
         get "/json_arr?select=data->0->>a&id=in.(5,6)" `shouldRespondWith`
-          [json| [{"a":"A"}, {"a":"[1,2,3]"}] |]
+          [json| [{"a":"A"}, {"a":"[1, 2, 3]"}] |]
```

Comment:
The test output changes slighty because `to_jsonb` adds a bit of whitespace.

Reply:
I see, and `to_jsonb` is used because of:

> -- Using to_jsonb instead of to_json to avoid missing operator errors when filtering:
> -- "operator does not exist: json = unknown"